src/pages/billing/Project.js

```diff
@@ -278,21 +288,39 @@ const ProjectDetail = ({ authorizeAndLoadAccounts, billingAccounts, billingProje
     exporting: { buttons: { contextButton: { x: -15 } } }
   }
 
-  const CostCard = ({ title, amount, ...props }) => {
+  const isProjectCostReady = projectCost !== null
+
+  const CostCard = ({ title, amount, showAsterisk, ...props }) => {
     return div({
       ...props,
       style: {
         ...Style.elements.card.container,
         backgroundColor: 'white',
         padding: undefined,
         boxShadow: undefined,
-        gridRowStart: 2
+        gridRowStart: '2'
       }
     }, [
-      div({ style: { flex: 'none', padding: '0.625rem 1.25rem' }, 'aria-live': projectCost !== null ? 'polite' : 'off', 'aria-atomic': true }, [
-        h3({ style: { fontSize: 16, color: colors.accent(), margin: '0.25rem 0.0rem', fontWeight: 'normal' } }, title),
-        div({ style: { fontSize: 32, height: 40, fontWeight: 'bold', gridRowStart: '2' } }, [amount])
-      ])
+      div(
+        {
+          style: { flex: 'none', padding: '0.625rem 1.25rem' },
+          'aria-live': projectCost !== null ? 'polite' : 'off',
+          'aria-atomic': true
+        },
+        [
+          div({ style: { fontSize: 16, color: colors.accent(), margin: '0.25rem 0.0rem', fontWeight: 'normal' } }, [title]),
+          div({ style: { fontSize: 32, height: 40, fontWeight: 'bold', gridRowStart: '2' } }, [
+            amount,
+            (!!showAsterisk && isProjectCostReady) ? span(
```

Comment:
Nit: `showAsterisk` is a boolean, so I think having the `!!` in front just makes it harder to read.